grobid-core/src/main/java/org/grobid/core/jni/PythonEnvironmentConfig.java

```diff
@@ -69,7 +86,10 @@ public static PythonEnvironmentConfig getInstanceForVirtualEnv(String virtualEnv
             pythons = Files.find(
                 Paths.get(virtualEnv, "lib"),
                 1,
-                (path, attr) -> path.getFileName().toString().contains("python3")
+                (path, attr) -> (
+                    path.toFile().isDirectory()
```

Comment:
In conda the lib folder is full of stuff and I can see now why we need the filtering. Just looking for directories seem to make sense (it was picking up the wrong one otherwise).